_config.yml

```diff
@@ -101,18 +101,29 @@ defaults:
 include:
   # dotfiles are excluded by default
   - .well-known
+  - docs/accessibility-assessment-ial1-account-creation.pdf
+  - docs/accessibility-assessment-ial2-identity-verification-process.pdf
 
 exclude:
+  - assets/img/favicons
+  - assets/js/build/main.js.LICENSE.txt
+  - babel.config.json
+  - CONTRIBUTING.md
+  - docs
   - Gemfile*
+  - jest.config.js
+  - LICENSE.md
   - Makefile*
   - node_modules
+  - OLD_URLS.yml
+  - package-lock.json
   - package.json
+  - README.md
   - scripts
   - spec
-  - webpack.config.js
+  - svgo.config.js
```

Comment:
It occurs to me we'll still need to manually exclude JS configuration files from the root, which seem like they could be fairly common for our tooling. Not sure if there's a great solution for this aside from requiring to allowlist every root-directory file.